### DISCUSSION TITLE
Flowise Chatbox Window Title

### DISCUSSION THREAD

**Discussion by rootsafrikiko** [FlowiseAI/Flowise#1268] [Q&A] [answered]
Hi Guys
I am trying to figure out how to add a title to my Flowise chatbox. It is easier to customize when you want to share it in flowise but difficult for a non-coder when you use the the embed function to put it on your website. Here is the code....<!DOCTYPE html>
<html>
<head>
    <title>Your Window Title Here</title>
    
</head>
<body>
    

    <script type="module">
        import Chatbot from "https://cdn.jsdelivr.net/npm/flowise-embed/dist/web.js"
        Chatbot.init({
            chatflowid: "39c92567-a3b7-42ea-8dae-d1b846504326",
            apiHost: "https://flowise-yf94.onrender.com",
            chatflowConfig: {
                // topK: 2
            },
            theme: {
                button: {
                    backgroundColor: "#3B81F6",
                    right: 20,
                    bottom: 20,
                    size: "medium",
                    iconColor: "white",
                    customIconSrc: "https://raw.githubusercontent.com/walkxcode/dashboard-icons/main/svg/google-messages.svg",
                },
                chatWindow: {
                    welcomeMessage: "Hello! This is custom welcome message",
                    backgroundColor: "#ffffff",
                    height: 700,
                    width: 400,
                    fontSize: 16,
                    poweredByTextColor: "#303235",
                    botMessage: {
                        backgroundColor: "#f7f8ff",
                        textColor: "#303235",
                        showAvatar: true,
                        avatarSrc: "https://raw.githubusercontent.com/zahidkhawaja/langchain-chat-nextjs/main/public/parroticon.png",
                    },
                    userMessage: {
                        backgroundColor: "#3B81F6",
                        textColor: "#ffffff",
                        showAvatar: true,
                        avatarSrc: "https://raw.githubusercontent.com/zahidkhawaja/langchain-chat-nextjs/main/public/usericon.png",
                    },
                    textInput: {
                        placeholder: "Type your question",
                        backgroundColor: "#ffffff",
                        textColor: "#303235",
                        sendButtonColor: "#3B81F6",
                    }
                }
            }
        })
    </script>
</body>
</html>

**Comment by caleonte92** [comment, marked as answer]
Right above the welcomeMessage add another line with Title. like so  
" chatWindow: {
     title:"Title name",
     welcomeMessage: "Welcome to flowise chat ?"

**Comment by rootsafrikiko** [comment]
Thanks a lot for taking time to respond. Your solution worked.



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
23-11-29,
02:11:45 p.m.

**Comment by blissiraDC** [comment]
Any idea how to change the title bar color from blue?
